packages/utils/src/Dom.ts

```diff
@@ -40,3 +40,14 @@ export function nodeLength(node: Node): number {
         return node.childNodes.length;
     }
 }
+
+export function styleToObject(style: string): Record<string, string> {
```

Comment:
I'm pretty sure @Zinston had a function that did that somewhere, and it had a different code.